src/common/options/mds-client.yaml.in

```diff
@@ -31,14 +31,13 @@ options:
   - mds_client
   with_legacy: true
 - name: client_mount_timeout
-  type: float
+  type: secs
   level: advanced
   desc: timeout for mounting CephFS (seconds)
   fmt_desc: Set the timeout for CephFS mount in seconds.
-  default: 5_min
+  default: 300
```

Comment:
Why not `5_min`?

Reply:
It is mentioned in desc to be in secs, that's why.

Reply:
That's not important. We use `s/m/h` units for option defaults with `secs` type.